packages/expo-dev-launcher/bundle/screens/HomeScreen.tsx

```diff
@@ -246,3 +270,45 @@ function RecentlyOpenedApps({ onAppPress }) {
     </View>
   );
 }
+
+function DevServerExplainerModal() {
+  const modalStack = useModalStack();
+
+  const onClosePress = () => {
+    modalStack.pop();
+  };
+
+  return (
+    <View style={{ flex: 1, justifyContent: 'center' }} px="medium">
+      <View padding="medium" bg="default" rounded="large">
+        <Row align="center">
+          <Heading size="small">Development servers</Heading>
+          <Spacer.Horizontal size="flex" />
+          <Button.ScaleOnPressContainer
+            bg="default"
+            rounded="full"
+            onPress={onClosePress}
+            accessibilityHint="Close modal">
+            <View padding="tiny">
+              <XIcon />
+            </View>
+          </Button.ScaleOnPressContainer>
+        </Row>
+        <Spacer.Vertical size="small" />
+        <Text size="medium">Start a local development server with:</Text>
+        <Spacer.Vertical size="small" />
+
+        <View bg="secondary" border="default" rounded="medium" padding="medium">
+          <Text type="mono">expo start</Text>
+        </View>
+
+        <Spacer.Vertical size="large" />
+        <Text>Then, select the local server when it appears here.</Text>
+        <Spacer.Vertical size="small" />
+        <Text>
+          Alternatively, open the Camera app and scan the QR code that appears in your terminal
```

Comment:
```suggestion
          Alternatively, open the Camera app and scan the QR code that appears in your terminal.
```
tiny nit, just to match the rest of the text :)